portfolio/src/main/webapp/style.css

```diff
@@ -1,4 +1,10 @@
+body{
```

Comment:
nit: can you reformat to add a space between body and {? And also all the following css rules.